catalog/core/catalog-core-urlresourcereader/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -18,6 +18,11 @@
         <OCD name="URL Resource Reader"
          id="ddf.catalog.resource.impl.URLResourceReader">
                 <AD
+                description="Check the box if you want the Resource Reader to automatically follow server issued redirects (HTTP Response Code 300 series)"
+                name="Follow Server Redirects" id="followRedirects" required="true"
+                type="Boolean" default="true"/>
```

Comment:
To maintain the current behavior can you update this to by `false` / disabled by default?